template/v1/Dockerfile

```diff
@@ -90,7 +90,7 @@ ARG MAMBA_DOCKERFILE_ACTIVATE=1
 RUN sudo ln -s $(which python3) /usr/bin/python
 
 # Update npm version
-RUN npm i -g npm
+RUN RUN npm update -g npm
```

Comment:
duplicated "RUN" command

Reply:
Good point, I'm making change to reduce `RUN` command in both v1.10.2 and v1 template in this PR